Tools/Scripts/libraries/webkitscmpy/webkitscmpy/program/revert.py

```diff
@@ -75,18 +75,24 @@ def parser(cls, parser, loggers=None):
     @classmethod
     def get_commit_info(cls, args, repository, **kwargs):
         commit_objects = []
-        commits_to_revert = []
         commit_bugs = set()
```

Comment:
Feels like we can generalize this code pretty easily.

In general, this isn't creating separate lists for Bugzillas and radars, it's creating a separate list for each tracker we have. So `commit_bugs = set()` should really be `commit_issues = {}` where each key represents a tracker (perhaps `Tracker.name` as discussed in #24026)